.github/workflows/MeTTa-CI.yml

```diff
@@ -35,4 +35,4 @@ jobs:
     
     - name: Run Python tests
       working-directory: ./
-      run: python -m unittest discover -s utilities/
```

Comment:
I thought the `unittest` could be run to discover every test cases on it's own? Why don't you use that?

Reply:
Let's do that for now. I wonder if we can find a better way in the future.